### ISSUE TITLE
Wrong results for SVG in Embed and Object

### ISSUE THREAD

**Issue by CheariX** [canitrust/backend#61]
Hi,

we briefly talked about this at OWASP.
I think, there is an Issue in

https://canitrust.in/detail/67
and
https://canitrust.in/detail/68

According to our results, write access should be given from HD (Host Document) to ED (Embedded Document = SVG) using EEs (Emedding elements: `<object>, <embed>`).

You can find our evalution testbed on 
http://your-sop.com/

Select **ED: Scalable Vector Graphics (SVG)** and then ** EE: `<iframe> <object>` and `<embed>`**.
In my recent Chromium, it states **write access** (w) **yes**.
By hovering over **yes** you can see the source code that produces the results.
Basically, it loads an SVG containing a script that tries to write **yes** into HD via EE.

**Comment by tuyenee** [comment]
Hi @CheariX, 

Thank you for your feedback and your interesting SOP testbed. It seems like we had a mistake in the payload that we used for this test case. However, we need to investigate it in more details and we will get back to you once we test it properly, probably next week.

**Comment by tuyenee** [comment]
Hi @CheariX,

You were absolutely right. Access for Javascript in SVG embedded via the `object` and `embed` tags to the host document's DOM should be allowed, given that both, the SVG and the host document, are served from the same origin. We have fixed our test script and re-ran the test-cases. The outcome results have confirmed your theory.

The correct results have been deployed to the respective test-cases: [67](https://canitrust.in/detail/67) and [68](https://canitrust.in/detail/68).

Thank you again for pointing out this issue!
